RxExample/RxExample/Examples/GeolocationExample/GeolocationViewController.swift

```diff
@@ -14,23 +14,21 @@ import CoreLocation
 #endif
 
 private extension Reactive where Base: UILabel {
-    var driveCoordinates: UIBindingObserver<Base, CLLocationCoordinate2D> {
+    var coordinates: UIBindingObserver<Base, CLLocationCoordinate2D> {
         return UIBindingObserver(UIElement: base) { label, location in
             label.text = "Lat: \(location.latitude)\nLon: \(location.longitude)"
         }
     }
 }
 
 private extension Reactive where Base: UIView {
-    var driveAuthorization: UIBindingObserver<Base, Bool> {
-        return UIBindingObserver(UIElement: base) { view, authorized in
-            if authorized {
-                view.isHidden = true
-                view.superview?.sendSubview(toBack:view)
+    func subviewPresence(_ subview: UIView) -> UIBindingObserver<Base, Bool> {
+        return UIBindingObserver(UIElement: base) { view, show in
+            if !show {
+                subview.removeFromSuperview()
             }
             else {
-                view.isHidden = false
-                view.superview?.bringSubview(toFront:view)
```

Comment:
Actually I have simplified it a bit by:
- totally removing hacky `UIBindingObserver` and using `rx.hidden`
- imperatively adding a `noGeolocationView` as a subview in `viewDidLoad` while still keeping it as a separate view in Storyboard

I believe it satisfies both my goal to improve readability of the example and your goal to keep code as exact as possible :)